packages/colorpickers/src/styled/Colorpicker/StyledSliders.ts

```diff
@@ -23,15 +27,15 @@ export const StyledSliders = styled.div.attrs({
   & > * {
     position: absolute;
     width: 100%;
-    height: ${props => getTrackMargin(props.theme) * 2 + getTrackHeight(props.theme)}px;
+    height: ${props => getTrackMargin(props) * 2 + getTrackHeight(props)}px;
   }
 
   & > :first-child {
-    top: -${props => getTrackMargin(props.theme)}px;
+    top: -${props => getTrackMargin(props)}px;
```

Comment:
Not request for change, noticed this could use implicit argument passing with `-${getTrackMargin}px`.